.github/workflows/tests.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: |
           yarn test:int --coverage --verbose
         env:
-          DATABASE_URL: ${{ matrix.database == 'postgres' && 'postgres://postgres:postgres@localhost:5432/postgres' || '' }}
```

Comment:
This change allows us to create databases and simply have to append its name to the connection string to communicate with it and it also fallback to the default database (`postgres`) if used directly.